README.md

```diff
@@ -417,220 +592,31 @@ Add into the server configuration file's authorization section.
 
 ## Monitoring
 
-If the monitoring port is enabled, the server will run a lightweight http server that has several endpoints defined, **[/varz, /connz, /routez, /subsz]**. All endpoints return a JSON object.
```

Comment:
This is in the Monitoring section of the docs in order to reduce the size of the readme.